group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupConfig.java

```diff
@@ -57,13 +62,20 @@ public class GroupConfig extends AbstractConfig {
             GroupCoordinatorConfig.CONSUMER_GROUP_HEARTBEAT_INTERVAL_MS_DEFAULT,
             atLeast(1),
             MEDIUM,
-            GroupCoordinatorConfig.CONSUMER_GROUP_HEARTBEAT_INTERVAL_MS_DOC);
+            GroupCoordinatorConfig.CONSUMER_GROUP_HEARTBEAT_INTERVAL_MS_DOC)
+        .define(SHARE_RECORD_LOCK_DURATION_MS_CONFIG,
+            INT,
+            ShareGroupConfig.SHARE_GROUP_RECORD_LOCK_DURATION_MS_DEFAULT,
+            atLeast(1),
```

Comment:
Should it be atleast(1) or atleast(1000)?

Reply:
Thanks for the review. I was honestly trying to just be consistent with the other configs defined here. I think this `atleast(1)` here is just to confirm it's a positive value, otherwise the lower and upper constraints will always be validated in the validate method.

Reply:
I too think this should be atleast(1000) given `group.share.min.record.lock.duration.ms ` lies between 1000 and 3600000

Reply:
Yeah you. have to be consistent with values defined in the KIP.